static/app/views/replays/detail/network/details/sections.tsx

```diff
@@ -0,0 +1,148 @@
+          onClick={onScrollToRow}
+          size="xs"
+        />
+      </Fragment>
+    ),
+    [t('Type')]: item.op,
+    [t('Method')]: item.data?.method ?? '',
+    [t('Status Code')]: item.data?.statusCode ?? '',
+    [t('Request Body Size')]: (
+      <SizeTooltip>{formatBytesBase10(item.data?.request?.size ?? 0)}</SizeTooltip>
+    ),
+    [t('Response Body Size')]: (
+      <SizeTooltip>{formatBytesBase10(item.data?.response?.size ?? 0)}</SizeTooltip>
+    ),
+    [t('Duration')]: `${(endMs - startMs).toFixed(2)}ms`,
+    [t('Timestamp')]: (
+      <TimestampButton
+        format="mm:ss.SSS"
+        onClick={(event: MouseEvent) => {
+          event.stopPropagation();
+          handleClick(item);
+        }}
+        startTimestampMs={startTimestampMs}
+        timestampMs={startMs}
+      />
+    ),
+  };
+
+  return (
+    <SectionItem title={t('General')}>
+      {keyValueTableOrNotFound(data, t('Missing request details'))}
+    </SectionItem>
+  );
+}
+
+export function RequestHeadersSection({item}: SectionProps) {
+  return (
+    <SectionItem title={t('Request Headers')}>
+      {keyValueTableOrNotFound(item.data.request?.headers, t('Headers not captured'))}
+    </SectionItem>
+  );
+}
+
+export function ResponseHeadersSection({item}: SectionProps) {
+  return (
+    <SectionItem title={t('Response Headers')}>
+      {keyValueTableOrNotFound(item.data.request?.headers, t('Headers not captured'))}
+    </SectionItem>
+  );
+}
+
+export function QueryParamsSection({item}: SectionProps) {
+  const queryParams = queryString.parse(item.description?.split('?')?.[1] ?? '');
+  return (
+    <SectionItem title={t('Query String Parameters')}>
+      <Indent>
+        <ObjectInspector data={queryParams} expandLevel={3} />
+      </Indent>
+    </SectionItem>
+  );
+}
+
+export function RequestPayloadSection({item}: SectionProps) {
+  const hasRequest = 'request' in item.data;
+  return (
+    <SectionItem
+      title={t('Request Payload')}
+      titleExtra={
+        <SizeTooltip>
+          {t('Size:')} {formatBytesBase10(item.data.request?.size ?? 0)}
+        </SizeTooltip>
+      }
+    >
+      <Indent>
+        <Warning warnings={item.data?.request?.warnings} />
+        {hasRequest ? (
+          <ObjectInspector data={item.data?.request?.body} expandLevel={2} />
+        ) : (
+          tct('Request body not found.', item.data)
+        )}
+      </Indent>
+    </SectionItem>
+  );
+}
+
+export function ResponsePayloadSection({item}: SectionProps) {
+  const hasResponse = 'response' in item.data;
+
+  return (
+    <SectionItem
+      title={t('Response Body')}
+      titleExtra={
+        <SizeTooltip>
+          {t('Size:')} {formatBytesBase10(item.data.response?.size ?? 0)}
+        </SizeTooltip>
+      }
+    >
+      <Indent>
+        <Warning warnings={item.data?.request?.warnings} />
+        {hasResponse ? (
+          <ObjectInspector data={item.data?.response?.body} expandLevel={2} />
```

Comment:
We're setting `expandLevel={2}` (down from 3) to try and make it quicker to switch tabs. 

See: https://github.com/getsentry/sentry/issues/46428